src/rest.cpp

```diff
@@ -246,9 +246,12 @@ static bool rest_headers(const CoreContext& context,
         return true;
     }
     case RetFormat::JSON: {
+        const NodeContext* const node = GetNodeContext(context, req);
+        if (!node || !node->llmq_ctx) return false;
```

Comment:
should return RESTERR, such as:
```
return RESTERR(req, HTTP_INTERNAL_ERROR, "no chainlock information");
```

Reply:
Added `GetLLMQContext` that will return a `HTTP_INTERNAL_SERVER_ERROR` if `LLMQContext` isn't initialized. Resolved in latest push!